Tasks/SeleniumWebDriverInstaller/task.json

```diff
@@ -0,0 +1,107 @@
+{
+    "id": "2C65196A-54FD-4A02-9BE8-D9D1837B7112",
+    "name": "SeleniumWebdriverInstaller",
+    "friendlyName": "Selenium WebDriver Installer",
+    "description": "Acquires and caches WebDriver for various browsers to run Selenium tests.",
+    "helpMarkDown": "[Supported versions of drivers](https://seleniumwebdrivers.azureedge.net/supportedversions/SupportedWebdrivers.json)",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "minimumAgentVersion": "2.103.0",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "preview": true,
+    "demands": [],
+    "instanceNameFormat": "Install Selenium WebDriver",
+    "inputs": [
+        {
+            "name": "chromeDriver",
+            "type": "boolean",
+            "label": "Chrome Driver",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Acquires the WebDriver for Chrome from <a href=\"https://sites.google.com/a/chromium.org/chromedriver/downloads\">here</a>. The driver can be referenced using the environment variable ChromeWebDriver."
+        },
+        {
+            "name": "chromeDriverVersion",
+            "type": "string",
+            "label": "Version",
```

Comment:
talked to prachi offline. we should add Chrome Driver Version etc in the disapl text

Reply:
+1 on this, please ensure you add this before merge